panels/config.py

```diff
@@ -11,7 +11,13 @@
 c.EVENT_BOOKED = {'colspan': 0}
 c.EVENT_OPEN   = {'colspan': 1}
 
-c.PANEL_ROOMS = [getattr(c, room.upper()) for room in c.PANEL_ROOMS]
+invalid_panel_rooms = [room for room in c.PANEL_ROOMS if not getattr(c, room.upper(), None)]
+
+for room in invalid_panel_rooms:
+    log.warning('panels plugin: panels_room config problem: '
+                'Ignoring "' + room.upper() + '" because it was not also found in [[event_location]] section.')
+
+c.PANEL_ROOMS = [getattr(c, room.upper()) for room in c.PANEL_ROOMS if room not in invalid_panel_rooms]
```

Comment:
@EliAndrewC let me know if you think there might be a more compact way to do this that's also as readable, I couldn't figure out how to get a list comprehension to exclude an invalid entry while also emitting a warning message on items we wanted to exclude.